ui/src/app/applications/components/applications-list/applications-summary.tsx

```diff
@@ -40,7 +40,7 @@ export const ApplicationsSummary = ({applications}: {applications: models.Applic
         },
         {
             title: 'CLUSTERS:',
-            value: new Set(applications.map(app => app.spec.destination.server)).size
+            value: new Set(applications.map(app => app.spec.destination.server)).size + new Set(applications.map(app => app.spec.destination.name)).size
```

Comment:
Hm, I think this would also not reflect the actual count either. 

If there's a mix of `server` and `name` destinations, the first `Set` would contain an entry for empty server (`""`), and the second set an entry for an empty name (`""`). So, in the worst case the count would be off by two. In the best case, it would be off by one (as it is now).

Reply:
thx - You're right - will filter this out. 
There is also an option that one app will use `name`, different will use `server` but both pointing to the same cluster... this will also result in a wrong result.

Reply:
@pasha-codefresh I wasn't able to prepare a local dev environment for argocd so I've left the issue :(

Reply:
@lukpep